src/main/java/apoc/load/LoadXls.java

```diff
@@ -20,7 +24,7 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
-import static apoc.util.Util.cleanUrl;
+import static apoc.util.Util.*;
```

Comment:
please also update the procedure `@Description`

Reply:
Ok